pyvista/utilities/helpers.py

```diff
@@ -505,7 +505,7 @@ def is_inside_bounds(point, bounds):
     """
     if isinstance(point, (int, float)):
         point = [point]
-    if isinstance(point, collections.Iterable) and not isinstance(point, collections.deque):
+    if isinstance(point, collections.abc.Iterable) and not isinstance(point, collections.deque):
```

Comment:
Any reason why we're not using a `Sequence` here?

Reply:
The only reason is that I didn't notice the `len(point)` check... sorry for being sloppy. I thought that it was only passed on to `deque`, which would be happy with an iterable. Although semantically speaking even without the `len` it should be preferable to have a sequence.